lib/Sub/Meta/Library.pm

```diff
@@ -46,10 +39,7 @@ sub register_list {
 
 sub get {
     my ($class, $sub) = @_;
-
-    unless (ref $sub && ref $sub eq 'CODE') {
-        _croak "required coderef: $sub";
-    }
+    return unless $Callable->check($sub);
```

Comment:
For convenience of `CreatorFunction#find_submeta`

https://github.com/kfly8/p5-Sub-Meta/blob/ba1e7070fae6dc42f60d780881e21ee0a23092f6/lib/Sub/Meta/CreatorFunction.pm#L20-L22